CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Unreleased
 
 - Add missing validation for the `context_lines` option and fix its behavior when passing `null` to make it working as described in the documentation (#1003)
+- Trim path from anonymous class in `Stacktrace` (#1016)
```

Comment:
Maybe it's worth mentioning somehow that the path is trimmed down according to the `prefixes` option. WDYT?

Reply:
Agree